easybuild/easyconfigs/c/Coreutils/Coreutils-8.23-GCC-4.9.2.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'ConfigureMake'
+
+name = "Coreutils"
```

Comment:
minimal required version for something else, maybe?

or just because we can, there doesn't need to be a reason ;)